Makefile

```diff
@@ -227,3 +227,6 @@ ggml.o: ggml.c ggml.h
 
 rwkv.o: rwkv.cpp rwkv.h
 	$(CXX) $(CXXFLAGS) -c rwkv.cpp -o rwkv.o
+
+librwkv.so: rwkv.o ggml.o
```

Comment:
Will it work when building on Windows?

Reply:
I should note that when buliding through python via scikit-build the user doesn't have to do anything special, inside the `SKBUILD` if block in the root CMakeLists.txt the build decides whether to build using cmake or just directly invoke the existing Makefile based on the OS. The `SKBUILD` check should also ensure that the CMakeLists.txt should still work if you just want to build the project as a library but don't care about the python package.